pkg/util/workqueue/prometheus/prometheus.go

```diff
@@ -25,13 +25,106 @@ import (
 // Package prometheus sets the workqueue DefaultMetricsFactory to produce
 // prometheus metrics. To use this package, you just have to import it.
 
+// Metrics subsystem and keys used by the workqueue.
+const (
+	WorkQueueSubsystem         = "workqueue"
+	DepthKey                   = "depth"
+	AddsKey                    = "adds_total"
+	QueueLatencyKey            = "queue_latency_microseconds"
+	WorkDurationKey            = "work_duration_microseconds"
+	UnfinishedWorkKey          = "unfinished_work_seconds"
+	LongestRunningProcessorKey = "longest_running_processor_microseconds"
```

Comment:
I think we should stick with one common unit for all the metrics for workqueue and not mix seconds and microseconds. Seconds is one the base units suggested in the prometheus docs (https://prometheus.io/docs/practices/naming/), so I think we can use that unless we have a good reason to use microseconds.

Reply:
Agree. Changed unit to seconds. PTAL